src/services/api/request.ts

```diff
@@ -36,47 +36,45 @@ const configuredKy = ky.create({
 // Generic Request Handler
 
 const defaultOptions: RequestOptions = {
-  kyOptions: {},
-  callbacks: {
-    setSubmitting: () => {}
-  },
-  accept: 'json'
+  accept: 'json',
+  method: 'get',
+  timeout: 30 * 1000,
+  slowResponseTimeout: 5 * 1000
 };
 
 export default async function request(
   route: string,
-  {
-    callbacks = defaultOptions.callbacks,
-    kyOptions = defaultOptions.kyOptions,
-    accept = defaultOptions.accept
-  }: RequestOptions = defaultOptions
+  options: RequestOptions = defaultOptions
 ): Promise<Response> {
   let response;
 
-  const mergedKyOptions = mergeKyOptions(kyOptions);
-  const { setErrors, setSubmitting } = callbacks;
+  const {
+    accept = defaultOptions.accept,
+    slowResponseTimeout = defaultOptions.slowResponseTimeout,
+    onSubmit,
+    onFinish,
```

Comment:
What would we use `onFinish` for?

Reply:
See an example [here](https://github.com/FixMyBerlin/fixmy.frontend/blob/a7f5deac9aaa5a5d07bf2c0227ae73714c77eac9/src/services/api/request.ts#L62). Actually you added this, I just renamed the callback.

Reply:
Oh, good thinking Sascha ;)